docs/src/tutorials/calibration.qmd

```diff
@@ -0,0 +1,94 @@
+As  a sharpness score of the forecaster, Kuleshov et al. proposed the average predicted variance
+
+$$sharpness(F_{1} ,\dots, F_T) = \frac{1}{T} \sum_{t=1}^T var{F_t}$$ \
+the smaller the sharpness, the tighter will be the confidence intervals on average.
+
+### Calibration Plots 
+
+To check the level of calibration, Kuleshov et al. proposed a calibration plot that display the true frequency of points in each confidence interval relative to the predicted fraction of points in that interval.\
+More formally, we choose $m$ confidence levels $0 ≤ p_1 < p_2 < . . . < p_m ≤ 1$; for each threshold $p_j$ , and compute the empirical frequency
+
+$$\hat{p}_j = \frac{|\{ y_t|F_t(y_t) \leq p_t, t= 1,2,\dots,T    \} |}{T}.$$
+
+To visualize the level of average calibration, we plot $\{(p_j,\hat{p_j}) \}_{j=1}^M$; A forecaster that is calibrated  will correspond a straight line on the plot that goes from $\{0,0\}$ to $\{1,1\}$ . \
+As a measure of the level of miscalibration of the forecaster, differently from what it is suggested in the original paper, we a measure  the area between the bisector of the plot and the line produced by the forecaster.
+
+ 
+### Post-training calibration
+As we  have said previously, uncertainty estimates obtained by deep BNNs tends to be miscalibrated. We introduced the support to a post-training technique for regression problems presented in  [Recalibration of Aleatoric and Epistemic Regression Uncertainty in Medical Imaging](https://arxiv.org/abs/2104.12376)
+by Max-Heinrich Laves, Sontje Ihler, Jacob F. Fast, Lüder A. Kahrs and Tobias Ortmaier and usually referred to as sigma-scaling. Using a Gaussian model, the technique consist in scaling the predicted standard deviation $\sigma$ with a scalar value $s$ to recalibrate the probability density function
+
+$$p(y|x; \hat{y}(x), \hat{σ}^2(x)) = \mathbb{N}( y; \hat{y}(x),(s  \cdot  \hat{σ}(x))^2 ).$$ 
+
+This results in the following minimization objective:
+
+$$L_G(s) = m \log(s) + \frac{1}{2}s^{−2} \sum_{i=1}^m (\hat{σ}^{(i)}_θ)^{−2} || y^{(i)} − \hat{y}_{\theta}^{(i)}||^2.$$
+
+In general, this equation can be optimized  respect to s with fixed values for the parameters $\theta$ using gradient descent in a second phase over a separate calibration set.  However, for the case of a gaussian distribution, the analytical solution is known  and takes the closed form
+
+$$s = \pm \sqrt{\frac{1}{m} \sum_{i=1}^m (\hat{σ}^{(i)}_θ)^{−2} || y^{(i)} − \hat{y}_{\theta}^{(i)}||^2}.$$
+
+Once the scalar $s$ is computed, all we have to do to obtain better calibrated predictions is to multiply the predicted standard deviation with the scalar.
+
```

Comment:
Might need some explanation here as to how this differs from Empirical Bayes (currently use to tune the prior, including observational noise in the regression case).